packages/framework/fluid-static/src/test/types/validateFluidStaticPrevious.generated.ts

```diff
@@ -142,20 +116,6 @@ declare function use_old_InterfaceDeclaration_IConnection(
 use_old_InterfaceDeclaration_IConnection(
     get_current_InterfaceDeclaration_IConnection());
 
-/*
- * Validate forward compatibility by using the old type in place of the current type.
- * If this test starts failing, it indicates a change that is not forward compatible.
- * To acknowledge the breaking change, add the following to package.json under
- * typeValidation.broken:
- * "InterfaceDeclaration_IFluidContainer": {"forwardCompat": false}
- */
-declare function get_old_InterfaceDeclaration_IFluidContainer():
-    TypeOnly<old.IFluidContainer>;
-declare function use_current_InterfaceDeclaration_IFluidContainer(
-    use: TypeOnly<current.IFluidContainer>): void;
-use_current_InterfaceDeclaration_IFluidContainer(
-    get_old_InterfaceDeclaration_IFluidContainer());
-
```

Comment:
I skimmed a couple of other packages that lost type tests like this and in most cases I see they were alpha APIs, but this and IFluidContainerEvents below are public; feels like something's wrong here.

Reply:
@CraigMacomber / @tylerbutler, expected changes?

Reply:
Yes. IFluidContainer is `@sealed` so adding items to it should not be a breaking change.

Reply:
Note it still includes tests to ensure functionality from it is not removed (the backwards compat tests)